Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor

```diff
@@ -117,7 +117,7 @@
     {
         @if ( ShowColumnChooser )
         {
-            <Field Display="Blazorise.Display.InlineBlock" Margin="Blazorise.Margin.IsAuto.OnY.Is2.FromStart">

```

Comment:
I think we should only remove Y margin, and leave the rest `Margin="Blazorise.Margin.Is2.FromStart"`

Reply:
With from start it looks like:
![image](https://github.com/user-attachments/assets/cccafe65-d2da-4675-82f7-32d481b8bacb)

So I keep it on both sides. 
![image](https://github.com/user-attachments/assets/0ee17904-743f-4d06-9f09-561d5ad93607)

Reply:
It seems to me that the BS4 is now broken.

![image](https://github.com/user-attachments/assets/55f8192a-4f3e-4197-947c-7b007f3b21c3)

vs live demo (https://bootstrapdemo.blazorise.com/tests/datagrid/column-chooser)

![image](https://github.com/user-attachments/assets/9950d46f-2ad3-43ae-9c42-be8904ac0c7d)

---

Same for BS5. Haven't look at other providers.

Reply:
The column chooser button needs different margins on different positions, thus I made it so.